libcxx/test/std/utilities/debugging/is_debugger_present.assertion.pass.cpp

```diff
@@ -0,0 +1,35 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20, c++23
+// REQUIRES: linux && no-filesystem
+
+// <debugging>
+
+// bool is_debugger_present() noexcept;
```

Comment:
Can you add comment that this test tests the "true" result and that `libcxx/test/std/utilities/debugging/is_debugger_present.pass.cpp` tests the properties and the `false` result. The same for the other tests.